docs/developer-docs/web-apps/custom-domains/dns-setup.mdx

```diff
@@ -12,109 +12,116 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+Now, you are all set to register your custom domain with the boundary nodes, and you can continue with step 2 of the [general custom domains instructions](using-custom-domains#custom-domains-on-the-boundary-nodes).
 
 ## GoDaddy
 
-In the following, it is explained how you can configure your domain on GoDaddy to
-be used to point to a canister hosted on the Internet Computer. As an illustration,
-this guide shows how to configure the domain `ic-domain.online` and the subdomain `example.ic-domain.online`
-to point to the canister with the ID `y5jqt-wqaaa-aaaam-abcoq-cai`.
+Below are the steps to configure the apex of a domain and
+a subdomain on GoDaddy to be used as a custom domain for a canister deployed on ICP.
 
 - #### Step 1: After purchasing your domain on GoDaddy, open your account and navigate to **My Products**.
 
 - #### Step 2: Click on the **DNS** button next to the domain:
 
-    ![Domain Management Overview](godaddy-overview.png)
+    ![Domain Management Overview](godaddy-overview.png)
 
 ### Apex
 
-Unfortunately, GoDaddy does not support configuring a `CNAME` record (or one of its alternatives, `ALIAS` or `ANAME`) for the apex of the domain, you need to make use of a workaround.
+Unfortunately, GoDaddy does not support configuring a `CNAME` record (or one of its alternatives, `ALIAS` or `ANAME`) for the apex of the domain; you need to make use of a workaround.
 
-  There are mainly two approaches:
+  There are two options:
 
-  - [_recommended_: Relying on a different DNS provider (e.g., Cloudflare).](#rely-on-an-alternative-dns-provider-recommended)
-  - [Directly configuring the IP addresses of the boundary nodes.](#directly-configure-the-ip-addresses)
+  - [**Recommended**: Relying on a different DNS provider (e.g., Cloudflare).](#rely-on-an-alternative-dns-provider-recommended)
+  - [Directly configuring the IP addresses of the boundary nodes.](#directly-configure-the-ip-addresses)
 
 Directly configuring the IP addresses is simpler compared to relying on another
 DNS provider, but is less resilient and performant. Therefore, it is advised you to use
 a different DNS provider (e.g., Cloudflare or Route 53).
 
 #### Rely on an alternative DNS provider (_recommended_)
 
-It is explained in this approach using Cloudflare as DNS provider. It works similar
+It is explained in this approach using Cloudflare as a DNS provider. It works similar
 with any other DNS provider that supports `CNAME`, `ALIAS`, or `ANAME` records
 for the apex of a domain.
 
-  - #### Step 1: Create a free account with Cloudflare, click on **Add site** in the top bar of the dashboard.
+  - #### Step 1: Create a free account with Cloudflare; click on **Add site** in the top bar of the dashboard.
 
-  - #### Step 2: Enter your domain (e.g., `ic-domain.online`) and click **Add site**.
+  - #### Step 2: Enter your domain (e.g., `ic-domain.live`) and click **Add site**.
```

Comment:
```suggestion
  - #### Step 2: Enter your domain (e.g., `ic-domain.online`) and click **Add site**.
```
Why change the domain name? All the screenshots are for `ic-domain.online` and not `ic-domain.live`.

Reply:
It is confusing to use two different example domains. We should be consistent in the examples used. Can we update the screenshots to all be consistent, or is there a benefit to using two different example domains?

Reply:
Updated screenshots